docs/swagger/spec.yml

```diff
@@ -20,8 +20,12 @@ consumes:
 tags:
```

Comment:
may want to add a filter by Band as well, but that's not something we currently have on the backend.